e2e/assets/assetscanister/src/e2e_project_frontend/assets/.ic-assets.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "match": "ignored.txt",
+    "ignore": true
+  },
+  {
+    "match": "*",
+    "cache": {
+      "max_age": 500
+    },
+    "headers": {
+      "x-header": "x-value"
+    }
+  },
+  {
+    "match": ".*",
```

Comment:
Can we target this at a specific file or directory?  One interpretation of ".*" as a regex is "match any character, 0 or more times."  But here I think we mean "match a dot, followed by anything or nothing."

So I'm also I'm wondering, and would like to see the test demonstrate: which is the behavior of the glob library (and therefore dfx)?

Reply:
> But here I think we mean "match a dot, followed by anything or nothing."

correct

We're currently limited until this issue is resolved https://github.com/BurntSushi/ripgrep/issues/2229
Once resolved, the user will be able to use `.` as regex when `ignore` field won't be set, and as a "match a dot" if `ignore: false`